### ISSUE TITLE
Include "to wit chapter" in NOIPM referral letters

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#273]
**Blocked by #271** 

### Business Context:

**As a** New Orleans referral letter
**I want** to include the "to wit chapter"
**So that** I can throw more shade on the conduct of the given officer than before

### In Scope

1. Make sure we have the latest referral letter template (pull from production database) since Christian now has the ability to tweak it
2. Tweak letter to include to wit chapter - see examples below

#### Text examples:
For allegations with Rule and Paragraph: 
"Rule 2: Moral Conduct: Par. 6 Unauthorized Force"

For allegations with Rule, Paragraph, and to wit Chapter: 
"Rule 2: Moral Conduct: Par. 6 Unauthorized Force: to wit NOPD Ch. 1.3 Use of Force"

For allegations with Rule, Paragraph, to wit Chapter, and Moral Conduct Violation: 
"Rule 2: Moral Conduct: Par. 6 Unauthorized Force: to wit NOPD Ch. 1.3 Use of Force, Par. 1-5"

### Out of Scope

1. Modifying the allegation manager to add the chapter - to be done on #271 
2.
3.

### Open Questions

1.
2.
3.

## Acceptance Criteria

1. Scenario:
   Given I have added a chapter on the allegation
   When I edit a letter that has an allegation that includes chapter 
   Then I see the text "to wit" in the letter similar to the examples

2. Scenario:
   Given
   When
   Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)

- [ ] Ensure to include all necessary mockups to this card.
- [ ] Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
- [ ] Discuss Feature Flag: Where could a development flag go to flag off incomplete work? Do we want to keep the flag long-term?
- [ ] If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous cards in the sequence have been finished.
- [ ] Have we added edge cases as ACs that we're considered from a QA perspective
- [ ] Should the new functionalities be offered only to users with certain permissions?
- [ ] How should this functionality differ between different orgs, and if it does differ, what is the hingepoint we should use for that functionality?

**Notes:**

**Comment by andrew-rist-tw** [comment]
the line should then be {rule} followed by {paragraph} followed by {chapter} followed by {directive} leaving out any that don't exist

**Comment by jordanngtw** [comment]
Old:
<img width="979" alt="Screenshot 2023-06-07 at 3 57 35 PM" src="https://github.com/PublicDataWorks/complaint-manager/assets/86383902/a08e5d3f-77ce-4027-b07c-495e086adaa1">

New:

<img width="1038" alt="Screenshot 2023-06-07 at 4 01 31 PM" src="https://github.com/PublicDataWorks/complaint-manager/assets/86383902/51418c08-0b57-4257-844f-151dcc643547">
